administrator/components/com_admin/sql/updates/postgresql/5.2.0-2024-06-01.sql

```diff
@@ -0,0 +1,35 @@
+ALTER TABLE "#__modules" ADD COLUMN "menu_assignment" TEXT NULL DEFAULT NULL AFTER `params`;
```

Comment:
```suggestion
ALTER TABLE "#__modules" ADD COLUMN "menu_assignment" text /** CAN FAIL **/;
```
1. Be consistent with how we do it elsewhere and use the /** CAN FAIL **/ installer hint.
2. PostgreSQL does not support `AFTER`. You should have researched that in their documentation. Anyway the order of columns in a table doesn't really matter.